addons/spreadsheet/static/src/assets_backend/helpers.js

```diff
@@ -1,9 +1,10 @@
-import { loadBundle } from "@web/core/assets";
+import { loadBundle, loadJS } from "@web/core/assets";
 
 /**
  * Load external libraries required for o-spreadsheet
  * @returns {Promise<void>}
  */
 export async function loadSpreadsheetDependencies() {
     await loadBundle("web.chartjs_lib");
+    await loadJS("/spreadsheet/static/lib/chartjs-chart-geo/chartjs-chart-geo.js");
```

Comment:
that's funny.
We could just `loadBundle("spreadsheet.dependencies")`, right ?
Or even include `spreadsheet.dependencies` in `spreadsheet.o_spreadsheet` bundle directly if we are fetching those libs in all cases